akvo/rsr/views/__init__.py

```diff
@@ -14,3 +14,10 @@ def index(request):
     """Redirect user to project directory or My RSR."""
 
     return HttpResponseRedirect(reverse('project-directory', args=[]))
+
+
+def lockpass(request):
```

Comment:
I think a little documentation here would help future developers understand what's going on.

From reading this, one wouldn't know why it exists and what makes it show the password-protected page.